Arithmetic/MultiplicationOfThreeFactors.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace CSharp_Practice.Arithmetic
+{
+    public class MultiplicationOfThreeFactors
+    {
+        public static void MultiplyThreeFactors()
+        {
+            Console.WriteLine("Input 3 numbers and it would multiple all three numbers for the result");
+            int firstNum = InputInteger();
+            int secondNum = InputInteger();
+            int thirdNum = InputInteger();
+
+            int result = firstNum * secondNum * thirdNum;
+
+            Console.WriteLine("{0}", result);
+        }
+
+        static int InputInteger()
```

Comment:
Yes, like this maybe add error handling and more string filtering.. and maybe put stuff like this into a base class so every exercise class can use them